libraries/freertos_plus/aws/greengrass/test/aws_test_ggd_unit.c

```diff
@@ -0,0 +1,586 @@
+/*
+ * FreeRTOS Greengrass V2.0.1
```

Comment:
The file naming seems inconsistent for this new file (`aws_tests_ggd_unit.c`) and the existing file `aws_tests_greengrass_discovery.c`. Can you update them to be consistent and also use the `_system` suffix in the old tests' file?

Reply:
Updated